source/lib/cdk-textract-stack.ts

```diff
@@ -1,4 +1,5 @@
-  /**********************************************************************************************************************
+//@ts-nocheck
+/**********************************************************************************************************************
  *  Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.                                           *
```

Comment:
We have over 200 resources in this stack which is a hard limit for Cloudformation resources. We would probably need to move the custom resources to a nested stack or to the Client stack